src/_premake_init.lua

```diff
@@ -268,6 +286,22 @@
 	}
 
 	api.register {
+		name = "debugtoolargs",
+		scope = "config",
+		kind = "list:string",
+		tokens = true,
+		pathVars = true,
+	}
+
+	api.register {
+		name = "debugtoolcommand",
+		scope = "config",
+		kind = "path",
+		tokens = true,
+		pathVars = true,
+	}
```

Comment:
Could be just debugtool() then? Or make it debuggerpath() and use it in conjunction with debugger()? So we get the tool's executable name from debugger() and combine it with debuggerpath()?